autorag/nodes/generator/run.py

```diff
@@ -35,7 +35,7 @@ def run_generator_node(modules: List[Callable],
     node_dir = os.path.join(node_line_dir, "generator")  # node name
     if not os.path.exists(node_dir):
         os.makedirs(node_dir)
-    qa_data = pd.read_parquet(os.path.join(project_dir, "data", "qa.parquet"))
+    qa_data = pd.read_parquet(os.path.join(project_dir, "data", "qa.parquet"),engine='pyarrow')
```

Comment:
spacing 🤣

Reply:
Yeah, please refactor python file before commit.

You can set this at commit tab setting at the right under corner in the pycharm.